tests/cross_fw/test_templates/template_test_weights_compression.py

```diff
@@ -128,3 +127,39 @@ def test_mixed_precision(self, mode, all_layers, ratio, ref_ids):
             dataset=dataset,
         )
         self.check_weights(compressed_model, ref_ids)
+
+    @staticmethod
+    @abstractmethod
+    def get_model_for_test_scale_estimation():
+        """
+        Returns a backend model for test_scale_estimation.
+        """
+
+    @staticmethod
+    @abstractmethod
+    def get_scale_estimation_ref():
+        """
+        Returns the reference output of calculate_quantization_params of ScaleEstimation.
+        """
+
+    def test_scale_estimation(self, mocker):
+        calc_q_params_spy = mocker.spy(ScaleEstimation, "calculate_quantization_params")
+        model = self.get_model_for_test_scale_estimation()
+
+        # prepare dataset with one input tensor
+        input = np.arange(0, 32 * 32, dtype=np.float32).reshape(1, 32, 32)
+        input[0, 15] *= 100  # make one channel relatively higher.
```

Comment:
I would expect this to be reflected in the references, but I don't see anything outstanding. Maybe the value should be higher.

Reply:
I think that this test intends to check the difference between torch and OV backends. It does not aim to check the algorithm's correctness. However, I agree that your proposal is good. I can add a new test which will check the error after quantization and demonstrate that the outlier channel has the lowest error against others

Reply:
Are you going to add this new test in the follow-up PR?